src/iio_wrapper.cpp

```diff
@@ -61,6 +61,57 @@ IIOWrapper::IIOWrapper()
 
     m_channel_temp = iio_device_find_channel(m_dev, "temp", false);
 
+    double valueScaleAccel_x;
+    iio_channel_attr_read_double(m_channel_accel_x, "scale", &valueScaleAccel_x);
+    m_fvalScaleAccel_x = valueScaleAccel_x;
+
+    double valueScaleAccel_y;
+    iio_channel_attr_read_double(m_channel_accel_y, "scale", &valueScaleAccel_y);
```

Comment:
Is it necessary to create new variables than passing the address of the final ones?

Reply:
The new variable are double and it passed to member variable that is float. An implicit conversion is made.